regsync.yaml

```
@@ -10179,6 +10227,12 @@ sync:
 - source: quay.io/cilium/operator-generic:v1.9.8
   target: registry.suse.com/rancher/mirrored-cilium-operator-generic:v1.9.8
   type: image
+- source: quay.io/cilium/operator-generic:v1.17.5
+  target: docker.io/rancher/quay.io/cilium/operator-generic:v1.17.5
+  type: image
+- source: quay.io/cilium/operator-generic:v1.17.5
+  target: registry.suse.com/rancher/quay.io/cilium/operator-generic:v1.17.5
```

```suggestion
  target: registry.suse.com/rancher/mirrored-cilium-operator-generic:v1.17.5
```
@adamkpickering
I think we want to use the existing target path here, maybe a bug in the image-mirror scripting?